sdk/resourcemanager/azure-resourcemanager-appservice/pom.xml

```diff
@@ -49,6 +49,7 @@
       --add-opens com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.arm=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.storage/com.azure.resourcemanager.storage=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
why do we skip revapi check for appservice and other non-storage services?

Reply:
They depends on storage.